WinEvent.Tests.ps1

```diff
@@ -0,0 +1,189 @@
+﻿    <#
+    .NOTES
+       
+#>
+
+Import-Module '.\DSCResources\MSFT_xWinEventLog\MSFT_xWinEventLog.psm1' -Prefix WinEventLog -Force
```

Comment:
It's better to use `$PSScriptRoot\DSCResources\MSFT_xWinEventLog\MSFT_xWinEventLog.psm1`, because it will allow run tests from other folders besides root folder of module.